### PR TITLE
Script for visual comparison of two correctionlibs

### PR DESCRIPTION
**Pull request by harrypuuter**

### CODE REVIEW

File: scripts/correctionlib_comparison.py [hunk 1]
```diff
@@ -0,0 +1,401 @@
...
+import matplotlib
+import matplotlib.pyplot as plt
+import mplhep as hep
+import numpy as np
+
+matplotlib.use("Agg")
+hep.style.use("CMS")
+
+parser = argparse.ArgumentParser(description="Plotting comparison of two correction libs")
+parser.add_argument("--input_a", type=str, help="json.gz correctionlib file A")
```
Comment (harrypuuter) [inline scripts/correctionlib_comparison.py:18]: --input-a without underscores in the args, same for all other args

File: scripts/correctionlib_comparison.py [hunk 2]
```diff
@@ -0,0 +1,401 @@
...
+parser.add_argument("--input_b", type=str, help="json.gz correctionlib file B")
+parser.add_argument("--tag_a", type=str, help="Name (str) A", default="A")
```
Comment (harrypuuter) [inline scripts/correctionlib_comparison.py:20]: add a bit more verbose help to signal that this is the name to be plotted

**Review by harrypuuter — COMMENTED** [review]
Very nice ! I like this a lot, and works really well 

One little change, when printing the plotting status on the terminal, could you add a printout of the correction that is currently plotted? Rest is very fine (even with typing :))

**Review by harrypuuter — COMMENTED** [review]
👍🏼

File: scripts/correctionlib_comparison.py (hunk 1 above)
Comment (a-monsch) [reply scripts/correctionlib_comparison.py]: Added in [0887179](https://github.com/KIT-CMS/TauAnalysis-CROWN/commit/0887179c5d37b3f287597f9342d1aacb647ce35d)

https://github.com/KIT-CMS/TauAnalysis-CROWN/blob/0887179c5d37b3f287597f9342d1aacb647ce35d/scripts/correctionlib_comparison.py#L17-L29

File: scripts/correctionlib_comparison.py (hunk 2 above)
Comment (a-monsch) [reply scripts/correctionlib_comparison.py]: Added in [0887179](https://github.com/KIT-CMS/TauAnalysis-CROWN/commit/0887179c5d37b3f287597f9342d1aacb647ce35d)

https://github.com/KIT-CMS/TauAnalysis-CROWN/blob/0887179c5d37b3f287597f9342d1aacb647ce35d/scripts/correctionlib_comparison.py#L17-L29

Comment (a-monsch) [reply scripts/correctionlib_comparison.py]: Also in the same commit: a more descriptive output:

https://github.com/KIT-CMS/TauAnalysis-CROWN/blob/0887179c5d37b3f287597f9342d1aacb647ce35d/scripts/correctionlib_comparison.py#L407-L414

**Review by a-monsch — APPROVED** [review]

**Pull request merged**